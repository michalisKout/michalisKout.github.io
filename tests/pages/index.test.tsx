import { render } from "@testing-library/react";
import App from "../../pages";

describe("Home page", () => {
  it("renders Home page", () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});
