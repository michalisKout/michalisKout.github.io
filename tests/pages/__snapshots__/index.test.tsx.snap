// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page renders Home page 1`] = `
<DocumentFragment>
  <main
    class=""
  >
    <section
      class="user--section"
    >
      <div
        class="user--inner-container"
      >
        <div
          class="user--inner-wrapper"
        >
          <div
            class="lg:w-1/2"
          >
            <h1
              class="user--title"
            >
              Who I am
            </h1>
            <section
              class="user--content"
            >
              <p>
                Hello I am Michael, Frontend engineer. I mainly develop React Applications focusing on quality deliverables. I am a big fan of clean code principals, clean architecture and best practises. I like researching and playing around with new front end technologies and techniques that can increase my productivity and improve time management. Finally, music is an integral part of my daily life and helps me relax and stay focused.
              </p>
            </section>
            <div
              class="user--social-medial--container"
            >
              <a
                aria-label="Twitter"
                class="mx-2"
                href="#"
              />
              <a
                aria-label="Facebook"
                class="mx-2"
                href="#"
              />
              <a
                aria-label="Linkden"
                class="mx-2"
                href="#"
              />
              <a
                aria-label="Github"
                class="mx-2"
                href="#"
              />
            </div>
          </div>
          <div
            class="user--image-container"
          >
            <div
              class="user--image-wrapper"
            >
              <div
                class="max-w-lg"
              >
                <img
                  alt=""
                  class="user--image"
                  src="//images.ctfassets.net/66v9jygob9x8/4wi7f5z2zjh9XIA1yWFo1N/49dd67b5a2c96ff3af0c0edd70322cc9/0981.jpg"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section
      class="info--container"
    >
      <div
        class="info--wrapper"
      >
        <div
          class="lg:w-1/2"
        >
          <div
            class="info--image"
            style="background-image: url(https://images.unsplash.com/photo-1593642532400-2682810df593?ixlib=rb-1.2.1&ixid=MXwxMjA3fDF8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=750&q=80);"
          />
        </div>
        <div
          class="info--content--wrapper"
        >
          <h2
            class="info--title"
          >
            Build Your New
            <span
              class="info--tag"
            >
              Idea
            </span>
          </h2>
          <p
            class="info--content"
          >
            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quidem modi reprehenderit vitae exercitationem aliquid dolores ullam temporibus enim expedita aperiam mollitia iure consectetur dicta tenetur, porro consequuntur saepe accusantium consequatur.
          </p>
          <div
            class="mt-8"
          >
            <a
              class="info--button"
              href="#"
            >
              Start Now
            </a>
          </div>
        </div>
      </div>
    </section>
    <footer
      class="footer"
    >
      <div
        class="container py-6"
      >
        <h1
          class="footer--title"
        >
          Join 31,000+ other and never miss 
          <br />
           out on new tips, tutorials, and more.
        </h1>
        <div
          class="flex justify-center mt-6"
        >
          <div
            class="subscribe--container"
          >
            <div
              class="subscribe--wrapper"
            >
              <input
                aria-label="Enter your email"
                class="subscribe--input"
                placeholder="Enter your email"
                type="email"
              />
              <button
                class="button-primary"
              >
                subscribe
              </button>
            </div>
          </div>
        </div>
        <hr
          class="breaking-line"
        />
        <div
          class="footer--menu"
        >
          <div>
            <a
              class="link-primary"
              href="#"
            >
              Brand
            </a>
          </div>
          <div
            class="footer--menu-wrapper"
          >
            <div
              class="-mx-4"
            >
              <span
                class="footer--menu--link"
                data-testid="nav-item"
              >
                <a
                  href="/"
                >
                  Home
                </a>
              </span>
              <span
                class="footer--menu--link"
                data-testid="nav-item"
              >
                <a
                  href="/portfolio"
                >
                  Portfolio
                </a>
              </span>
              <span
                class="footer--menu--link"
                data-testid="nav-item"
              >
                <a
                  href="/experience"
                >
                  Experience
                </a>
              </span>
              <span
                class="footer--menu--link"
                data-testid="nav-item"
              >
                <a
                  href="/contact"
                >
                  Contact
                </a>
              </span>
            </div>
          </div>
        </div>
      </div>
    </footer>
  </main>
</DocumentFragment>
`;
